test/test_schema.rb

```diff
@@ -369,7 +397,7 @@ def self.resolve_type(_type, _obj, _ctx)
     end
 
     assert_raises ArgumentError do
-      GraphQL::Client::Schema.generate(schema)
+      GraphQL::Client::Schema.generate(schema, raise_on_unknown_enum_value: true)
```

Comment:
```suggestion
      GraphQL::Client::Schema.generate(schema)
```

Similarly, if this code works the same as it used too, I'd like to leave it as it was.